pages/workspaces/[workspaceId]/repository-insights/[insightId]/activity.tsx

```diff
@@ -98,7 +97,7 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   }
 
   // Keeping this here so we are sure the page is not private before we fetch the social card.
-  const ogImage = await fetchSocialCard(`insights/${insightId}`);
```

Comment:
`fetchSocialCard` isn't used anywhere now so the file, https://github.com/open-sauced/app/blob/95d545f6119f0f971afdd12445890aeaf2a06e06/lib/utils/fetch-social-card.ts, for it can be removed from the project.

Reply:
Good catch. Deleted